task2

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func main() {
+	nums := []int{1234, 6, 0, 3, 4}
+
+	sort.Slice(nums, func(i, j int) bool {
```

Comment:
sort.Slice(nums, func(i, j int) bool {
		return nums[i] < nums[j]
	})
	max := nums[0] + nums[1] + nums[2] + nums[3]
	min := nums[1] + nums[2] + nums[3] + nums[4]
	
	
that need to move to different func